pkg/controller/controller_binding.go

```diff
@@ -349,38 +352,67 @@ func isPlanBindable(serviceClass *v1alpha1.ServiceClass, plan *v1alpha1.ServiceP
 
 func (c *controller) injectBinding(binding *v1alpha1.Binding, credentials map[string]interface{}) error {
 	glog.V(5).Infof("Creating Secret %v/%v", binding.Namespace, binding.Spec.SecretName)
```

Comment:
Again, nor the comment, nor the name of the method was chosen by me, but I'm happy to rename it. Also, after https://github.com/kubernetes-incubator/service-catalog/issues/994 is done, the only code left in this method will be upserting secret, so could rename method to `upsertSecret` or something like that afterwards.

For now just will update the comment.